l10n/en/firefox/welcome/page20-21.ftl

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/welcome/20/
+### URL: https://www-dev.allizom.org/firefox/welcome/21/
+
+welcome-page20-21-old-version = You’re on an older version of { -brand-name-firefox }
+
+# Page 20 header
+welcome-page20-21-your-firefox = Your { -brand-name-firefox } may start to have problems on January 14, 2025
```

Comment:
What's the l01n plan for these pages? Is it only going through Smartling or do the new Fluent files need to be added to the pontoon config for community l10n as well?

It might be nice to include en-GB as a translation so we can format the dates with the day first, but not sure how best to handle that if not through Pontoon. Maybe we can just not worry about it.

Reply:
So AFAIK and based on my conversation with @slightlyoffbeat, we're going to be pinging the l10n team that this PR should take priority for translations once it's merged, so I'm assuming the l10n team will be letting their translators know